src/reducers/reducers.js

```diff
@@ -15,17 +15,18 @@ function selectedClassroom(state = {}, action ){
   }
 }
 
-function classrooms(state = {
+function allClassrooms(state = {
   isFetching: false,
-  didInvalidate: false,
   items: []
 }, action) {
   switch (action.type) {
     case types.REQUEST_CLASSROOMS:
+      console.log('REDUCERS: REQUEST_CLASSROOMS')
       return Object.assign({}, state, {
         isFetching: true
       })
     case types.RECEIVE_CLASSROOMS:
+      console.log('REDUCERS: RECEIVE_CLASSROOMS')
       return Object.assign({}, state, {
         isFetching: false,
         items: action.classrooms
```

Comment:
Actually this doesn't matter because I realised that the object structure is wrong so `items` will be replaced.